src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/infoboxbuilder/SpecialInfoboxBuilderPageObject.java

```diff
@@ -0,0 +1,281 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.special.infoboxbuilder;
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.elements.oasis.pages.TemplatePage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.PortableInfoboxPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialPageObject;
+import com.wikia.webdriver.testcases.infoboxbuilder.InfoboxBuilderTests;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.testcases.infoboxbuilder.InfoboxBuilderTests'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)